app/controllers/email_alert_subscriptions_controller.rb

```diff
@@ -72,7 +72,9 @@ def email_alert_signup_api
   end
 
   def subscriber_list_title
-    EmailAlertTitleBuilder.call(
+    title_builder = EmailAlertTitleBuilder
+    title_builder = EmailAlertListTitleBuilder if signup_presenter.combine_mode == "or"
```

Comment:
Could this be written using a ternary operator?

```suggestion
    title_builder = signup_presenter.combine_mode == "or" ? EmailAlertListTitleBuilder : EmailAlertTitleBuilder
```